docs/fundamentals/diagnostics/sample/src/DiagDemo/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
```

Comment:
launchBrowser is still missing for both kestrel and web, and there should also be a URL field.